Packs/GoogleCloudLogging/Integrations/GoogleCloudLogging/GoogleCloudLogging.py

```diff
@@ -0,0 +1,285 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+import httplib2
+import urllib3
+from typing import Dict, Any
+from google.oauth2 import service_account
+from apiclient import discovery
+import google_auth_httplib2
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+SCOPES = ['https://www.googleapis.com/auth/logging.read']
+
+''' CLIENT CLASS '''
+
+
+class GoogleCloudLoggingClient(BaseClient):
+    """
+    A Client class to wrap the google cloud api library.
+    """
+
+    def __init__(self, client_secret: str, proxy: bool,
+                 insecure: bool):
+        """
+        :param client_secret: A string of the credentials.json generated
+        :param proxy:
+        :param insecure:
+        """
+        try:
+            credentials = service_account.Credentials.from_service_account_info(info=client_secret, scopes=SCOPES)
+            if proxy or insecure:
+                http_client = google_auth_httplib2.AuthorizedHttp(
+                    credentials, http=self.get_http_client_with_proxy(proxy, insecure))
+                self.service = discovery.build('logging', 'v2', http=http_client)
+            else:
+                self.service = discovery.build('logging', 'v2', credentials=credentials)
+        except Exception as e:
+            raise e
+
+    # disable-secrets-detection-start
+
+    @staticmethod
+    def get_http_client_with_proxy(proxy, insecure):
+        """
+        Create an http client with proxy with whom to use when using a proxy.
+        :param proxy: Whether to use a proxy.
+        :param insecure: Whether to disable ssl and use an insecure connection.
+        :return:
+        """
+        if proxy:
```

Comment:
Why are you not using the `GSuiteClient` from the `GsuitepiModule`?
See `GsuiteAuditor` integration for example